pkgs/applications/audio/mpdevil/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, fetchFromGitHub
+, python3Packages
+, gdk-pixbuf, glib, gobject-introspection, gtk3
+, intltool
+, wrapGAppsHook }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mpdevil";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "SoongNoonien";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0l7mqv7ys05al2hds4icb32hf14fqi3n7b0f5v1yx54cbl9cqfap";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection gtk3 intltool wrapGAppsHook
```

Comment:
```suggestion
    gobject-introspection gtk3 intltool wrapGAppsHook glib.dev
```

Reply:
@SuperSandro2000 sometimes you recommand lib.getDev but not here, why is that ?